<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMeshHighlightClass</class>
 <widget class="QMainWindow" name="CMeshHighlightClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshHighlight</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QVTKWidget" name="VTKWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenMesh"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionDeleteLitter"/>
   <addaction name="separator"/>
   <addaction name="actionRepairHole"/>
   <addaction name="actionReconstruction"/>
   <addaction name="actionSutureMesh"/>
   <addaction name="actionParameterization"/>
   <addaction name="actionBakeTexture"/>
   <addaction name="actionShowMeshIndices"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MeshHighlight.qrc">
     <normaloff>:/MeshHighlight/Icon/save.png</normaloff>:/MeshHighlight/Icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDeleteLitter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MeshHighlight.qrc">
     <normaloff>:/MeshHighlight/Icon/delete.png</normaloff>:/MeshHighlight/Icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRepairHole">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/Hole.png</normaloff>Icon/Hole.png</iconset>
   </property>
   <property name="text">
    <string>RepairHole</string>
   </property>
  </action>
  <action name="actionBakeTexture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/Bake.png</normaloff>Icon/Bake.png</iconset>
   </property>
   <property name="text">
    <string>BakeTexture</string>
   </property>
   <property name="toolTip">
    <string>BakeTexture</string>
   </property>
  </action>
  <action name="actionReconstruction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/Reconstruction.png</normaloff>Icon/Reconstruction.png</iconset>
   </property>
   <property name="text">
    <string>Reconstruction</string>
   </property>
  </action>
  <action name="actionOpenMesh">
   <property name="icon">
    <iconset>
     <normaloff>Icon/OpenMesh.png</normaloff>Icon/OpenMesh.png</iconset>
   </property>
   <property name="text">
    <string>OpenMesh</string>
   </property>
  </action>
  <action name="actionParameterization">
   <property name="icon">
    <iconset resource="MeshHighlight.qrc">
     <normaloff>:/MeshHighlight/Icon/Reconstructor.png</normaloff>:/MeshHighlight/Icon/Reconstructor.png</iconset>
   </property>
   <property name="text">
    <string>Parameterization</string>
   </property>
  </action>
  <action name="actionSutureMesh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="MeshHighlight.qrc">
     <normaloff>:/MeshHighlight/Icon/regionGrowing.png</normaloff>:/MeshHighlight/Icon/regionGrowing.png</iconset>
   </property>
   <property name="text">
    <string>SutureMesh</string>
   </property>
  </action>
  <action name="actionShowMeshIndices">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MeshHighlight.qrc">
     <normaloff>:/MeshHighlight/Icon/circleAreaPicking.png</normaloff>:/MeshHighlight/Icon/circleAreaPicking.png</iconset>
   </property>
   <property name="text">
    <string>ShowMeshIndices</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MeshHighlight.qrc"/>
 </resources>
 <connections/>
</ui>
